vars/main.yml

```diff
@@ -5,7 +5,10 @@
 # See tasks/{RedHat,Debian,Ubuntu}.yml
 openio_repository_creds: "{% if repo.value.user is defined and repo.value.password is defined %}{{ repo.value.user }}:{{ repo.value.password }}@{% endif %}"
 
-openio_repository_mirror_url_base: "http://{{ openio_repository_creds }}mirror.openio.io/pub/repo/openio"
-openio_repository_mirror_url_base_nocreds: "http://mirror.openio.io/pub/repo/openio"
+openio_repository_mirror_host: "mirror.openio.io"
+openio_repository_mirror_url_base: "http://{{ openio_repository_creds }}{{ openio_repository_mirror_host }}/pub/repo/openio"
```

Comment:
I suppose we can change the path `/pub/repo/openio` to a parameter aswell.

Reply:
Yes, I agree, we will need this someday, but in another commit/PR, no need to mix things up